.eslintrc.yaml

```diff
@@ -0,0 +1,145 @@
+parser: "@typescript-eslint/parser"
+plugins:
+  - "@typescript-eslint"
+  - import
+parserOptions:
+  project: ./package.json
+  ecmaVersion: 6
+  sourceType: module
+  ecmaFeatures:
+    modules: true
+extends:
+  - eslint:recommended
+  - plugin:@typescript-eslint/eslint-recommended
+  - plugin:@typescript-eslint/recommended
+rules:
+  no-template-curly-in-string: off
+  quotes: off
+  "@typescript-eslint/interface-name-prefix": off
+  eol-last: off
+  arrow-parens:
+    - error
+    - "as-needed"
+  "@typescript-eslint/no-namespace": off
+  max-len:
+    - error
+    - code: 150
+  sort-keys: off
+  comma-dangle: off
```

Comment:
I would suggest enabling this rule with the `always-multiline` config, this ensures clean diffs.

See https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8